src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { Input } from 'reactstrap'
+import './ProfileModifier.css'
+
+export default function ProfileCard (props) {
+  const [month] = useState(parseInt(new Date().getMonth()) + 1)
```

Comment:
You don't need a hook here, and with my suggestion for the date function you can remove this entire line.